lib/pure/collections/intsets.nim

```diff
@@ -7,9 +7,10 @@
 #    distribution, for details about the copyright.
 #
 
-## Deprecated by the generic `PackedSet` for ordinal sparse sets.
-## **See also:**
-## * `Ordinal packed sets module <packedsets.html>`_ for more general packed sets
+## Deprecated in favor of the generic `packedsets module <packedsets.html>`_
+## for ordinal sparse sets.
+
+{.deprecated: "Use the 'packedsets' module instead".}
```

Comment:
@konsumlamm if this holds, compiler+stdlib should be patched to use packedsets instead of intsets otherwise rebuilding nim creates a lot of deprecation warnings:

> compiler/lookups.nim(13, 3) Warning: Use the 'packedsets' module instead; intsets is deprecated [Deprecated]

Reply:
Oh right, I didn't think about that.